### PR TITLE
Blog-APP: Authentication

### PR DESCRIPTION
**Pull request by RitobrotoMukherjee**
## Fixed errors sign up
- Added `:confirmable` in user model
- added `letter_opener`  gem for confirmation

### Project requirements I have performed
- Remove `current_user` method in `ApplicationController`, devise will provide us with one.
- [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
  - Basic wireframe design. Will design it well after approval. 
  - Can register a new user.
  - User logs in with a combination of email and password.
  - Hashed passwords should be stored in the database.
  - Ask for confirmation by email.
  - Can reset password.

## I have repeated the week, so I am alone in the team.

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
```
Comment (Munsa1) [inline app/views/devise/registrations/new.html.erb:1]: - [ ] When trying to sign up, I am getting the following error, kindly fix this issue.

image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @RitobrotoMukherjee, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/devise/registrations/new.html.erb (hunk 1 above)
Comment (RitobrotoMukherjee) [reply app/views/devise/registrations/new.html.erb:1]: @Munsa1 Thank you for your feedback. Actually, I only tested using the seeded user for the login. My bad. Let me fix it.

**Review by DammyShittu — APPROVED** [review]
Hi @RitobrotoMukherjee,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RitobrotoMukherjee** [comment]
@DammyShittu Thank you for approving my project :100:

**Pull request merged**